src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -645,6 +646,15 @@ public SymbolInfo GetSymbolInfo(CrefSyntax crefSyntax, CancellationToken cancell
                 : SymbolInfo.None;
         }
 
+        public SymbolInfo GetSymbolInfo(LocalFunctionStatementSyntax localFunctionSyntax, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            CheckSyntaxNode(localFunctionSyntax);
+
+            return CanGetSemanticInfo(localFunctionSyntax)
```

Comment:
won't this always return true for a LocalFunction?  If so, i'm not sure what purpose this serves.